nucypher/config/characters.py

```diff
@@ -39,32 +39,37 @@ class UrsulaConfiguration(CharacterConfiguration):
     CHARACTER_CLASS = Ursula
     NAME = CHARACTER_CLASS.__name__.lower()
 
-    DEFAULT_REST_HOST = '127.0.0.1'
     DEFAULT_REST_PORT = 9151
+    DEFAULT_DEVELOPMENT_REST_HOST = '127.0.0.1'
     DEFAULT_DEVELOPMENT_REST_PORT = 10151
     __DEFAULT_TLS_CURVE = ec.SECP384R1
-    DEFAULT_DB_NAME = '{}.db'.format(NAME)
+    DEFAULT_DB_NAME = f'{NAME}.db'
     DEFAULT_AVAILABILITY_CHECKS = False
     LOCAL_SIGNERS_ALLOWED = True
 
     def __init__(self,
+                 rest_host: str = None,
                  worker_address: str = None,
                  dev_mode: bool = False,
                  db_filepath: str = None,
-                 rest_host: str = None,
                  rest_port: int = None,
                  tls_curve: EllipticCurve = None,
                  certificate: Certificate = None,
                  availability_check: bool = None,
                  *args, **kwargs) -> None:
 
-        if not rest_port:
-            if dev_mode:
+        if dev_mode:
+            rest_host = self.DEFAULT_DEVELOPMENT_REST_HOST
```

Comment:
Do we want to check that `rest_host` isn't specified before overriding the value, similar to what we do for `rest_port`? i.e.

```suggestion
        if dev_mode:
            if not rest_host:
                rest_host = self.DEFAULT_DEVELOPMENT_REST_HOST
```

Reply:
:heavy_check_mark: